movethebytes/data-transfer/krakensync/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "krakensync"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = "0.1.58"
+anyhow = "1"
+env_logger = "0.9.0"
+if-addrs = "0.7.0"
+log = "0.4"
+rand = "0.8"
+serde_json = "1"
+soketto = "0.7.1"
+testground = "0.4.0"
+thiserror = "1"
+tokio = { version = "1", default-features = false, features = ["sync", "rt-multi-thread", "macros", "net"] }
+tokio-stream = { version = "0.1", default-features = false, features = [] }
+tokio-util = { version = "0.7", default-features = false, features = ["compat"] }
+futures = "0.3.1"
+rcgen = "0.10.0"
+rustls = { version = "0.20.7", default-features = false, features = ["quic"] }
+krakensync = { git = "https://github.com/n0-computer/krakensync", branch = "ramfox/pub" }
```

Comment:
we should get changes from `ramfox/pub` merged.

Reply:
I would actually like to punt on this until after the meeting and I've had a second to breath and look over the branch.